tools/openpilot_env.sh

```diff
@@ -1,5 +1,6 @@
 if [ -z "$OPENPILOT_ENV" ]; then
-  export PYTHONPATH="$HOME/openpilot:$PYTHONPATH"
+  OP_ROOT="$(dirname "$(dirname "${BASH_SOURCE[0]}")")"
```

Comment:
```suggestion
  OP_ROOT="$( cd $(dirname "$(dirname "${BASH_SOURCE[0]}")") && pwd )"
```

Think you need this to get the absolute path.

Reply:
Both work for me on Ubuntu 20.04 opening a new terminal window, is this suggestion safer?

Reply:
Not sure how it works for you. For me the `OP_ROOT` is jus equal to `.`

Reply:
Oh if you print `OP_ROOT` when sourcing from your .bashrc it gets the absolute path, but when you source or run the script outside `.bashrc` it's relative.